src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1312,7 +1318,7 @@ protected String checkJavaVersion(final PrintStream logger, String javaCommand,
     protected void openConnection(final TaskListener listener, final SlaveComputer computer) throws IOException, InterruptedException {
         PrintStream logger = listener.getLogger();
         logger.println(Messages.SSHLauncher_OpeningSSHConnection(getTimestamp(), host + ":" + port));
-        connection.setTCPNoDelay(true);
+        connection.setTCPNoDelay(!disableTcpNoDelay);
```

Comment:
I find the double negative here kind of confusing.

Reply:
agree, every thing sound weird and complex, I'm gonna re-think all again.